third_party/move/move-vm/transactional-tests/tests/tests.rs

```diff
@@ -65,24 +37,36 @@ static TEST_CONFIGS: Lazy<Vec<TestConfig>> = Lazy::new(|| {
     vec![
         TestConfig {
             name: "baseline",
-            runner: Arc::new(NoHooksRunner),
             experiments: &[],
             language_version: LanguageVersion::latest(),
             vm_config: vm_config_for_tests(VerifierConfig::production()),
             include: &[],
-            exclude: &["paranoid-tests"],
+            exclude: &["/paranoid-tests/"],
         },
         TestConfig {
             name: "paranoid-mode-only",
-            runner: Arc::new(SkipBytecodeVerifierRunner),
             experiments: &[("access-use-function-check", false)],
             language_version: LanguageVersion::latest(),
             // Verifier config is irrelevant here, because we disable verifier for these tests.
             // Importantly, paranoid checks are enabled.
-            vm_config: vm_config_for_tests(VerifierConfig::unbounded()),
+            vm_config: vm_config_for_tests(
+                VerifierConfig::unbounded().set_scope(VerificationScope::Nothing),
+            ),
             include: &["paranoid-tests"],
             exclude: &[],
         },
+        TestConfig {
+            name: "paranoid",
```

Comment:
Would be good to have a more descriptive name (as compared to the `paranoid-mode-only` above). Why cannot this be just part of `paranoid-mode-only` config above?

Reply:
The name will become part of the baseline file suffix, so I think long isn't very good. For that same reasonn, I don't want to use the above config (`foo.paranoid-only,exp` doesn't make sense if we also have `foo.baseline.exp`). Mayybe we can clean this up later and run the tests in `paranoid-only` also with bytecode verifier, for now I wanted to reduce the number of changes in this PR.